uitools/examples/qml_quick/src/demos/BasemapGalleryDemo.qml

```diff
@@ -72,5 +73,6 @@ DemoPage {
                 }
             }
         }
+        //! [Set up BasemapGallery QML]
```

Comment:
This snippet is QML API specific because of the `Connections` object. Because the `BasemapGallery` documentation is shared amongst the C++ and QML API this snippet may be confusing to the C++ API crowd.

We may have to reference both the C++ and QML APIs in the BasemapGallery documentation, and make it clear which one is for whom. @ldanzinger, thoughts?

Reply:
Hmm, not only the Connections object, but the declaration of the Map object as well ties this to the QML API... This is an issue in pretty much every snippet here... Maybe we just need some standard text before each code snippet that says something like "This snippet shows how to XYZ using the QML API. The workflow for using the Toolkit with the C++ API is similar, but the ArcGIS Runtime objects will be created in C++ code instead."

Not sure what other options we have. I don't think I'm a fan of showing QML, C++ Quick, and Widget snippets in each class doc - that is likely going to be overkill...

Reply:
I see, so then it might be better to show the C++ API which is more complicated (at least I found it that way when writing the demos), but it might be a bit more messy since we have to set up different snippets also on the C++ side. A good point would be to redirect the users to the sample pages from our repository.

Reply:
@ldanzinger It would be the Quick C++ API and the Quick QML API in the same class doc. There wouldn't be a widgets snippet. I don't know if that changes your feelings?